packages/truffle-core/lib/mnemonics/mnemonic.js

```diff
@@ -0,0 +1,73 @@
+/**
+* @module mnemonic;
+* @requires module:Configstore
+* @requires module:random
+* @requires module:bip39
+* @requires module:ethereumjs-wallet/hdkey
+*/
```

Comment:
Does your editor put these here automatically? This is awesome!

Reply:
What exactly are these?  I'm not familiar

Reply:
I wish! It does formatting to line the params up, but I put the docs in manually. I use Sublime right now but thinking about switching to Atom or Webstorm if they have better JS packages.

Reply:
@eggplantzzz just jsdocs, documentation for modules and functions. I used docs a lot when developing in PHP, its sort of habit for me to add them to new files!

Reply:
Thanks!